test/model/KDTreeTest.cpp

```diff
@@ -20,15 +20,15 @@ namespace kdtree {
     using testing::Pair;
     using Algorithm = PlaneSelectionAlgorithm::Algorithm;
 
-    class KDTreeTest : public ::testing::TestWithParam<std::tuple<std::vector<Array3>, std::vector<IndexArray3>,
-                Algorithm,
-                std::vector<Array3> > > {
+    class KDTreeTest : public ::testing::TestWithParam<std::tuple<std::vector<Array3>, std::vector<IndexVector>,
```

Comment:
Please fix the includes here:

`#include "../../src/KDTree/input/TetgenAdapter.h"` relative imports are OS dependant and can easily break. This should be depending on the CMake targets